mava/utils/make_env.py

```diff
@@ -94,14 +94,17 @@ def make_jaxmarl_env(env_name: str, config: Dict) -> Tuple[Environment, Environm
 
     kwargs = config["env"]["kwargs"]
     if "smax" in env_name.lower():
-        kwargs["scenario"] = map_name_to_scenario(config["env"]["scenario"])
+        kwargs["scenario"] = map_name_to_scenario(config["env"]["scenario"]["task_name"])
 
     # Placeholder for creating JAXMARL environment.
```

Comment:
Is this still just a 'placeholder'?

Reply:
Sorry, this is pointing to the wrong location. See line 99 below 👍🏻

Reply:
I updated the comment to be more informative